search-app/docs/api.md

```diff
@@ -239,9 +238,9 @@ The CORS headers are supported on search endpoints. Check [CORS Documentation](h
  * `page_num` - The page number to return
  * `offset` - As an alternative to page_num, a 0-based offset of individual results may be specified
  * `scroll` - A boolean flag (true/false) that allows all results to be retrieved efficiently. `page_size` is supported with `scroll` while `page_num` and `offset` are not. If `scroll` is `true` then the first call of a scroll session sets the page size; `page_size` is ignored on subsequent calls.
- * `sort_key` - Indicates one or more fields to sort on. Described below
- * `pretty` - return formatted results if set to true
- * `token` - specifies a user/guest token from EDL or Launchpad to use to authenticate yourself. EDL tokens can be specified as a header with the `Authorization: Bearer` prefix. Launchpad tokens can be specified as the header with `Authorization:`. 
+ * `sort_key` - Indicates one or more fields to sort on. Described below.
+ * `pretty` - Return formatted results if set to true.
+ * `token` - specifies a user/guest token from [EDL](https://urs.earthdata.nasa.gov/) or [Launchpad](https://launchpad.nasa.gov) for use as authentication. However this practice should be considered deprecated in favor of using the standard [Authorization header](#header).
```

Comment:
I don't think we want to include links to URS or Launchpad in this usage description. 1) we don't do so for other token references in the doc, 2) the links aren't actually to the token management API endpoints for either of URS or Launchpad and 1+2=3) client token management can be complicated and we shouldn't try to explain it here

Reply:
My apologies @jceaser for the confusion, but I don't think we want to add the "deprecation" language here. The token query parameter is fully supported functionality with no target for removal. Both train leadership and ESDIS have weighed in on it recently with the decision to simply keep it without modification. We can't unilaterally make the public claim that it's "deprecated".